components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.common/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

Comment:
Update the year range. Also please check the license header.

Reply:
Fixed issues in all the places - https://github.com/wso2/identity-api-server/pull/471/commits/716afd37c24161bdb757d5e2ddea5a111e753ece